typescript/cli/src/tests/warp/warp-rebalancer.e2e-test.ts

```diff
@@ -161,7 +161,7 @@ describe('hyperlane warp rebalancer e2e tests', async function () {
   afterEach(async () => {
     try {
       rmSync(REBALANCER_STRATEGY_CONFIG_PATH);
-    } catch (e) {
+    } catch (_e) {
       // Ignore
     }
```

Comment:
nice one, wdyt about the `force: true` option?

Reply:
Better with force